src/Symfony/Component/Security/Core/Signature/ExpiredSignatureStorage.php

```diff
@@ -9,14 +9,14 @@
  * file that was distributed with this source code.
  */
 
-namespace Symfony\Component\Security\Http\LoginLink;
+namespace Symfony\Component\Security\Core\Signature;
 
 use Psr\Cache\CacheItemPoolInterface;
 
 /**
- * @final
+ * @author Ryan Weaver <ryan@symfonycasts.com>
  */
-class ExpiredLoginLinkStorage
+final class ExpiredSignatureStorage
```

Comment:
This class has been introduced in 5.2 and is not experimental. We need a BC layer for this move, isn't it?

Reply:
This class was introduced as part of the login link package, which is only supported by the authenticator-based security. So I think it was just forgotten that this class is experimental. @weaverryan can you confirm here?

Reply:
Fine for me 👍 Shall we add the missing `experimental` flag to the new class then?